tests/unit/models/PermissionTest.php

```diff
@@ -83,7 +83,10 @@ public function testOwnershipSuccess()
     public function testOwnershipFailure()
     {
         // NB: Not 1 (superadmin).
-        $userId = 2;
+        //Create user.
+        $newPassword = createPassword();
+        $userName = \Yii::app()->securityManager->generateRandomString(8);
+        $userId = \User::insertUser($userName, $newPassword, 'John Doe', 1, $userName . '@mail.com');
```

Comment:
`@example.org` : it's done for sample and test.

Do not use an existing website (except `limesurvey.(coo|org)`)